scripts/m4t/audio_to_units/audio_to_units.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import logging
+import torch
+import torchaudio
+from seamless_communication.models.unit_extraction import UnitExtractor
+from seamless_communication.models.inference import Translator
+from seamless_communication.models.vocoder import load_vocoder_model, Vocoder
+from itertools import groupby
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def main():
+    parser = argparse.ArgumentParser(
```

Comment:
So the reason to convert back to audio is simply for verification? If we don't expect this as a common need maybe we could skip? Given we already abandoned earlier audio_synthesizer :)

Reply:
I'll just leave it as a convenience method in `UnitExtractor` then, which people can call and use. And remove it from here.